Makefile

```diff
@@ -10,7 +10,18 @@ setup:
 .env:
 	cp .env.dev .env
 
-dev: .env
+certs:
+	rm -f dev/certs/fake.*
+	openssl req -x509 \
+		-newkey rsa:2048 \
+		-days 365 \
+		-nodes \
+		-subj "/CN=localhost" -addext "subjectAltName=DNS:localhost,IP:127.0.0.1" \
+		-keyout dev/certs/fake.key \
+		-out dev/certs/fake.crt
+	cat dev/certs/fake.{key,crt} > dev/certs/fake.pem
```

Comment:
Let's always use a secure setup during development, for now only the liquidsoap inputs and the icecast stream are secure, but maybe we can also setup nginx to load the certificates.